src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -44,8 +44,17 @@ const FollowInfoRow = ({
           </Text>
         </View>
       </GlobalButton>
+      <View
+        style={{
+          height: 2,
+          width: 2,
+          backgroundColor: COLORS.black,
+          borderRadius: 999,
+          marginHorizontal: 8
+        }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded style here could be moved to the `StyleSheet.create` method for better organization and reusability. This also improves performance as the styles defined in `StyleSheet.create` are processed only once rather than every render.

```diff
-       <View
-         style={{
-           height: 2,
-           width: 2,
-           backgroundColor: COLORS.black,
-           borderRadius: 999,
-           marginHorizontal: 8
-         }}
-       />
+       <View style={styles.dotSeparator} />

// Add this to your StyleSheet.create
dotSeparator: {
  height: 2,
  width: 2,
  backgroundColor: COLORS.black,
  borderRadius: 999,
  marginHorizontal: 8
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->